Atividades/Priscila_Vieira/Exercicios.js

```diff
@@ -0,0 +1,168 @@
+// NOME: Priscila Baer Gomes Vieira
+
+class Animal{
+	constructor(nome, idade, cor, social){
+		this.nome = nome
+		this.idade = idade
+		this.cor = cor
+		this.consultas = []
+		this.social = social
+	}        
+	
+    consultar(){
+		this.consultas.unshift(new Date())
+	}
+  
+    brincar(){
+        console.log("Hora de brincar!!")
+    }
+	
+    acariciar(){
+		console.log("Hora do carinho")
+    }
+	
+    alimentar(comida){
+		console.log(`${this.nome} recebeu ${comida}`)
+	}
+}
+
+class Domesticado extends Animal{
+	
+	constructor(nome, idade, cor, social, castrado){
+		super(nome, idade, cor, social)
+		this.castrado = castrado
+		this.vacinas = []
+	}
+
+
+	vacinar(vacina){
+		this.vacinas.push(vacina)
+		super.consultar()
```

Comment:
Show que usou o super! No meu eu não usei, mas acho que o teu jeito faz mais sentido. O meu eu chamei consultar numa classe Animal, depois nas classes Cachorro, Gato etc eu chamei no construtor o this.consultas e nos métodos eu coloquei this.consultar(), não coloquei o super, mas vou fazer assim da próxima vez!